test/controllers/test_ontologies_controller.rb

```diff
@@ -282,11 +282,100 @@ def test_detach_a_view
     assert_equal onto["viewOf"], ont.id.to_s
   end
 
+  def test_ontology_agents
+    ontologies_and_submissions = create_ontologies_and_submissions(ont_count: 1, submission_count: 1, process_submission: true)
```

Comment:
you need to create two ontologies, to be sure that you get the agents of ontology, and not a mix